dts/bindings/input/microchip,cap12xx.yaml

```diff
@@ -10,6 +10,18 @@ include: i2c-device.yaml
 properties:
   int-gpios:
     type: phandle-array
+    description: |
+      interrupt gpio when not using polling mode.
+
+  repeat:
+    type: boolean
+    description: |
+      Property to enable the interrupt repeat mode for prolonged touch events.
+
+  poll-interval:
```

Comment:
`poll-interval-ms`, then you can tell the unit without having to refer to the binding doc